api/_mixins/HTMLHyperlinkElementUtils__HTMLAnchorElement.json

```diff
@@ -33,7 +33,7 @@
               "version_added": "14"
             },
             "safari": {
-              "version_added": "1"
+              "version_added": "3"
```

Comment:
For the most part I think the correct version is Safari 1, not 3. Can you test Safari 1 and 3 with the collector? If any of the members are supported in Safari 1, then it's the parent interface which is wrong.

Reply:
I don't have access to Safari 1-3 currently, but Safari 3 was originally set in #7267.

Reply:
https://github.com/mdn/browser-compat-data/pull/7267 says "using results from Safari 3-14" so I guess it could be earlier in some cases. I'll go through and suggest what I think is more likely in each case.